source/Octopus.Tentacle.Contracts/Observability/ITentacleClientObserver.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System;
+using Octopus.Diagnostics;
 
 namespace Octopus.Tentacle.Contracts.Observability
 {
     public interface ITentacleClientObserver
     {
-        void RpcCallCompleted(RpcCallMetrics rpcCallMetrics);
-        void UploadFileCompleted(ClientOperationMetrics clientOperationMetrics);
-        void DownloadFileCompleted(ClientOperationMetrics clientOperationMetrics);
-        void ExecuteScriptCompleted(ClientOperationMetrics clientOperationMetrics);
+        void RpcCallCompleted(RpcCallMetrics rpcCallMetrics, ILog logger);
```

Comment:
This will require a change in Server as the contract has changed to take an ILog